runtime/syntax/vim.vim

```diff
@@ -824,7 +831,7 @@ if !exists("skip_vim_syntax_inits")
   hi def link vimBehaveError	vimError
   hi def link vimCollClassErr	vimError
   hi def link vimErrSetting	vimError
-  hi def link vimEmbedError	Normal
+  hi def link vimEmbedError	vimError
```

Comment:
This (link to `Normal`) was actually a deliberate change by @justinmk four years ago (https://github.com/neovim/neovim/commit/1f1728e9bf27bdb2972d1b699e44bbdc8ec0ca1c). There was no rationale given, but I'd be happy to drop the change here if someone remembers why.

Reply:
Thinking about it, the thinking is probably since we intentionally _don't_ embed languages other than Lua, at least give them a normal coat of paint instead of blazing red hazard signs :)

Reply:
Performance.
We don't maintain this section so @justinmk shouldn't have changed this even if it slows down Neovim on vim files.